### PR TITLE
fix errors in .d.ts bundle

### PR DESCRIPTION
**Pull request by menduz**

### CODE REVIEW

**Review by gonpombo8 — APPROVED** [review]

File: packages/@dcl/ecs/src/systems/crdt/transports/rendererTransport.ts [hunk 1]
```diff
@@ -7,6 +7,13 @@ const componentIds = Object.values(ECSComponentIDs)
   .map(Number)
 
 export function createRendererTransport(): Transport {
+  if (typeof globalThis.dcl === 'undefined') {
+    // TODO: replace with new rpc
+    throw new Error(
```
Comment (leanmendoza) [inline packages/@dcl/ecs/src/systems/crdt/transports/rendererTransport.ts:12]: image

Comment (leanmendoza) [reply packages/@dcl/ecs/src/systems/crdt/transports/rendererTransport.ts:12]: change `globalThis.dcl` to just `dcl` as in `initialization.ts:17`

**Review by leanmendoza — APPROVED** [review]

File: packages/@dcl/ecs/src/systems/crdt/transports/rendererTransport.ts (hunk 1 above)
Comment (leanmendoza) [reply packages/@dcl/ecs/src/systems/crdt/transports/rendererTransport.ts:12]: I approved the PR, but this should be changed, I test modifying the `game.js` and it works perfectly!

File: packages/@dcl/ecs/src/systems/crdt/transports/rendererTransport.ts [hunk 2]
```diff
@@ -7,6 +7,13 @@ const componentIds = Object.values(ECSComponentIDs)
   .map(Number)
 
 export function createRendererTransport(): Transport {
+  if (typeof globalThis.dcl === 'undefined') {
```
Comment (leanmendoza) [inline packages/@dcl/ecs/src/systems/crdt/transports/rendererTransport.ts:10]:
```suggestion
  if (typeof dcl === 'undefined') {
```

**Pull request merged**